src/main/java/edu/bu/metcs673/trackr/user/TrackrUserController.java

```diff
@@ -13,93 +15,106 @@
 import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import edu.bu.metcs673.trackr.api.GenericApiResponse;
-import edu.bu.metcs673.trackr.common.CommonConstants;
-import edu.bu.metcs673.trackr.security.JWTUtil;
-import net.minidev.json.JSONObject;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletResponse;
+import javax.validation.Valid;
+
+import static edu.bu.metcs673.trackr.common.CommonConstants.JWT_COOKIE_MAX_AGE_MINUTES;
+import static edu.bu.metcs673.trackr.common.CommonConstants.JWT_COOKIE_NAME;
+import static edu.bu.metcs673.trackr.common.CommonConstants.JWT_COOKIE_PATH;
 
 /**
- * Controller for Users Management. Contains a 'Create' API for new users to
- * register with the application. Also contains an API which generates a new JWT
- * token for the user to use for authentication.
- * 
- * @author Tim Flucker
+ * Controller for Users Management. Contains a 'Create' API for new users to register with the
+ * application. Also contains an API which generates a new JWT token for the user to use for
+ * authentication.
  *
+ * @author Tim Flucker
  */
 @Validated
 @RestController
-@RequestMapping("/api/v1/users")
```

Comment:
why are we getting rid of this? I thought all of our APIs would follow this pattern?

Reply:
I wanted to create specific paths that are going to be unsecured. For `/api/v1/users` we want to keep it locked down. Trying to do principle of least privilege. If  `/api/v1/users` is unsecured then you  can do the following. 
- `GET /api/v1/users/1`
- `POST GET /api/v1/users/1`
- `DELTE GET /api/v1/users/1`
etc.